### ISSUE TITLE
os.PIDFD_NONBLOCK is missing, yet according to documentation it should be present 

### ISSUE THREAD

**Issue by n6koma7a** [python/cpython#127583]
# Bug report

### Bug description:

```python
# Add a code block here, if required
import os
os.PIDFD_NONBLOCK
```

https://docs.python.org/3/library/os.html#os.PIDFD_NONBLOCK

>  python3.13                                                                                                                                                                                                                                             Wed 04 Dec 2024 01:57:39 AM +03
Python 3.13.0 (main, Oct  8 2024, 00:00:00) [GCC 14.2.1 20240912 (Red Hat 14.2.1-3)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> os.PIDFD_NONBLOCK
Traceback (most recent call last):
  File "<python-input-1>", line 1, in <module>
    os.PIDFD_NONBLOCK
AttributeError: module 'os' has no attribute 'PIDFD_NONBLOCK'. Did you mean: 'EFD_NONBLOCK'?

### CPython versions tested on:

3.13

### Operating systems tested on:

Linux

**Comment by JelleZijlstra** [comment]
The docs say the constant is available on Linux >= 5.10.

However, I can replicate this on a Linux system running kernel 5.15 using current main:

```
$ uname -a
Linux devshared41 5.15.0-1070-aws #76~20.04.1-Ubuntu SMP Mon Sep 2 12:20:36 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
$ ./python -c 'import os; print(hasattr(os, "PIDFD_NONBLOCK"))'
False
```

The code looks correct: https://github.com/python/cpython/blob/12397a5781664bf43da98454db07cdfdec3ab815/Modules/posixmodule.c#L17410

And manpages confirm the constant should exist: https://man7.org/linux/man-pages/man2/pidfd_open.2.html

Maybe we're missing an `#include`?

cc @kumaraditya303 who added this in #93313

**Comment by stephen-hansen** [comment]
just browsing through here, looks like relevant kernel thread: https://lore.kernel.org/all/ZkrZb91EgZoaOybZ@t420/T/

Seems like the missing `#include` is on `<linux/pidfd.h>`.

**Comment by kumaraditya303** [comment]
Fixed by #127593
